pkg/querier/blocks_store_queryable.go

```diff
@@ -133,6 +134,10 @@ func newBlocksStoreQueryableMetrics(reg prometheus.Registerer) *blocksStoreQuery
 			Name: "cortex_querier_blocks_queried_total",
 			Help: "Number of blocks queried to satisfy query. Compared to blocks found, some blocks may have been filtered out thanks to query and compactor sharding.",
 		}),
+		compactedBlocksQueried: promauto.With(reg).NewCounter(prometheus.CounterOpts{
+			Name: "cortex_querier_compacted_blocks_queried_total",
```

Comment:
Why do we need to track this metric in the querier at all? Why not tracking this metric in the store-gateway? There's a quite a lot of wiring required to do to track this metric in the querier, when we could easily track it in the store-gateway.

Reply:
It's a good point. I was thinking we'd want to track this per querier so we could directly compare compacted blocks with the other `blocksStoreQueryableMetrics`, but `cortex_bucket_store_series_blocks_queried` makes sense too.